src/shared/helpers/converters.ts

```diff
@@ -17,7 +17,7 @@ export function convertEthToLiveTX(ethTX: EthTransaction): EthereumTransaction {
       ethTX.value !== undefined
         ? new BigNumber(ethTX.value.replace("0x", ""), 16)
         : new BigNumber(0),
-    recipient: eip55.encode(ethTX.to),
+    recipient: ethTX.to ? eip55.encode(ethTX.to) : "",
```

Comment:
I'd do something more along the lines of ` ethTX.to !==undefined ? eip55.encode(ethTX.to) : ""` to match the rest if it's fine for you

Minor